boot.php

```diff
@@ -17,7 +17,7 @@
  * easily as email does today.
  */
 
-require_once('include/autoloader.php');
+require_once(__DIR__ . DIRECTORY_SEPARATOR. 'vendor' . DIRECTORY_SEPARATOR . 'autoload.php');
```

Comment:
I like the idea of using `DIRECTORY_SEPARATOR`. It is way more cross-platform compatible.